doc/source/basics.rst

```diff
@@ -1788,6 +1788,46 @@ but occasionally has non-dates intermixed and you want to represent as missing.
 In addition, :meth:`~DataFrame.convert_objects` will attempt the *soft* conversion of any *object* dtypes, meaning that if all
```

Comment:
FYI ``.convert_objects`` was deprecated, so remove this. (you can instead show

``df.apply(pd.to_numeric)`` or ``df.apply(to_datetime)`` or ``df.apply(to_timedelta)`` (you don't need to actually show these, just explain that these are the replacements for ``.convert_objects``). maybe throw in a reference to ``.select_dtypes`` (which might be in a different section)